strongbox-commons/src/main/resources/application.yaml

```diff
@@ -62,4 +62,31 @@ management:
       exposure:
         include: health,info,beans,metrics,trace,scheduledtasks,threaddump
       base-path: /api/monitoring
-
+cacheManagerConfiguration:
+  groupConfig:
+    name: strongbox
+    password: password
+  enableMulticastConfig: true
```

Comment:
@cbrianhill multicast should be disabled by default

Reply:
Good catch, will disable this.  I'd flipped it for local testing.